lib/usb-device.js

```diff
@@ -16,7 +16,7 @@ class USBDevice {
   }
 
   // opens the device and sets up the endpoints for out object
-  async claimDevice () {
+  claimDevice () {
```

Comment:
Yeah. Bugs from not waiting for async functions are hard to debug, so just making all public calls have it, even if they don't do any promise work, helps alleviate some of that. That's what we've been doing, anyway. Internal functions don't need to be that way, if you trust the folks working on it (i.e., you!).